react-hot-button/my-app/src/HotButton.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react';
+import './App.css';
+
+export function HotButton() {
+  const [clicks, setClicks] = useState(0);
+  const [buttonColor, setButtonColor] = useState('black');
```

Comment:
Since the button color can be computed from the clicks, it's best to not have it be separate state. Just make it a normal variable and assign it the color you want.

Other than this, everything looks great!